README.md

```diff
@@ -1,12 +1,10 @@
 # Accessible autocomplete
 
----
-
-**We cannot currently respond to issues or pull requests.**
+The accessible autocomplete is a component that helps users choose countries and territories in answers.
```

Comment:
```suggestion
The accessible autocomplete is a component that helps users choose answers from a list we provide.
```

Reply:
Suggest `answers from a list you provide`